nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestCommentsMigrator.java

```diff
@@ -279,7 +240,135 @@ public void testProbe() {
         List<Statement> statementList = graph.getStatements(commentRes, null, null);
         graph.remove(statementList);
 
+        // No more relation comments detected as 'Relation'
         assertEquals(MIGRATION_STATE_PROPERTY, migrator.probeState());
+
+        // Migrate the created property based comment to secured
+        migrateFromPropertyToSecured(migrator);
+
+        // No more unsecured property comments
+        assertEquals(MIGRATION_STATE_SECURED, migrator.probeState());
+    }
+
+    protected static void sleep(long millis) {
+        try {
+            Thread.sleep(millis);
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * @since 11.1
+     */
+    protected void migrateFromPropertyToSecured(Migrator migrator) {
+        List<String> progressLines = new ArrayList<>();
+
+        runMigration(() -> migrator.run(MIGRATION_STEP_PROPERTY_TO_SECURED, getMigrationContext(progressLines)));
```

Comment:
`progressLines` usage looks weird to me. Previous usage was less surprising as the anonymous was directly in the test but as it isn't the case anymore I would rather go to a static class, inside this class, holding this list.